packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityStakeStep/CommunityStakeStep.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+
+import EnableStake from './EnableStake';
+import SignStakeTransactions from './SignStakeTransactions';
+
+import './CommunityStakeStep.scss';
+
+interface CommunityStakeStepProps {
+  onOptOutEnablingStake: () => void;
+  createdCommunityName: string;
+}
```

Comment:
would be cool if we could use a `types.ts` file in same folder

Reply:
same: per our conversation, we should create types.ts file if there is more than one type for file

Reply:
yup, no need to move this to types, resolving